components/support/rc_crypto/nss_sys/src/lib.rs

```diff
@@ -65,5 +63,19 @@ nss_exports! {
     unsafe fn NSS_IsInitialized() -> PRBool;
     unsafe fn NSS_GetVersion() -> *const c_char;
     unsafe fn NSS_VersionCheck(importedVersion: *const c_char) -> PRBool;
+    unsafe fn NSS_SecureMemcmp(ia: *const c_void, ib: *const c_void, n: usize) -> c_int;
     unsafe fn PK11_HashBuf(hashAlg: SECOidTag::Type, out: *mut c_uchar, r#in: *const c_uchar, len: PRInt32) -> SECStatus;
+    unsafe fn PK11_FreeSlot(slot: *mut PK11SlotInfo);
+    unsafe fn PK11_FreeSymKey(symKey: *mut PK11SymKey);
+    unsafe fn PK11_DestroyContext(context: *mut PK11Context, freeit: PRBool);
+    unsafe fn PK11_GetInternalSlot() -> *mut PK11SlotInfo;
+    unsafe fn PK11_ImportSymKey(slot: *mut PK11SlotInfo, r#type: CK_MECHANISM_TYPE, origin: PK11Origin::Type, operation: CK_ATTRIBUTE_TYPE, key: *mut SECItem, wincx: *mut c_void) -> *mut PK11SymKey;
```

Comment:
We could say `key: *const SECItem` to make things closer to how they are in reality, but it's probably better to keep this mirroring the real declaration, even if it's not const-correct.